src/contexts/StringContext.tsx

```diff
@@ -0,0 +1,84 @@
+import { createContext, useContext } from 'react'
+
+export interface WizardStrings {
+    reviewLabel: string
+    unknownError: string
+    errorString: string
+    stepsAriaLabel: string
+    contentAriaLabel: string
+    actionAriaLabel: string
+    detailsAriaLabel: string
+    sortableMoveItemUpAriaLabel: string
+    sortableMoveItemDownAriaLabel: string
+    removeItemAriaLabel: string
+    deselectAllAriaLabel: string
+    selectAllAriaLabel: string
+    clearButtonTooltip: string
+    pasteButtonTooltip: string
+    backButtonText: string
+    cancelButtonText: string
+    nextButtonText: string
+    fixValidationErrorsMsg: string
+    submitText: string
+    submittingText: string
+    internalFormFieldGroupError: string
+    moreInfo: string
+    hideSecretTooltip: string
+    showSecretTooltip: string
+    spinnerButtonTooltip: string
+    syncButtonTooltip: string
+    isRequired: string
+    required: string
+    getEnterPlaceholderString: string
+    getSelectPlaceholderString: string
+    getCollapsedPlaceholderString: string
+    getAddPlaceholderString: string
+    expandToFixValidationErrors: string
+    keyPathRequired: string
+    keyedValueError: string
+}
+
+export const defaultStrings: WizardStrings = {
+    reviewLabel: 'Review',
+    unknownError: 'Unknown Error',
+    errorString: 'error',
+    stepsAriaLabel: 'steps',
+    contentAriaLabel: 'content',
+    actionAriaLabel: 'Action',
+    detailsAriaLabel: 'Details',
+    sortableMoveItemUpAriaLabel: 'Move item up',
+    sortableMoveItemDownAriaLabel: 'Move item down',
+    removeItemAriaLabel: 'Remove item',
+    deselectAllAriaLabel: 'Deselect all',
+    selectAllAriaLabel: 'Select all',
+    clearButtonTooltip: 'Clear',
+    pasteButtonTooltip: 'Paste',
+    backButtonText: 'Back',
+    cancelButtonText: 'Cancel',
+    nextButtonText: 'Next',
+    fixValidationErrorsMsg: 'Please fix validation errors',
+    submitText: 'Submit',
+    submittingText: 'Submitting',
+    internalFormFieldGroupError:
+        'toggleAriaLabel or the titleText prop FormfieldGroupHeader is required to make the toggle button accessible',
+    moreInfo: 'More info',
+    hideSecretTooltip: 'Hide secret',
+    showSecretTooltip: 'Show secret',
+    spinnerButtonTooltip: 'Loading',
+    syncButtonTooltip: 'Refresh',
+    isRequired: 'is required',
```

Comment:
We should not be manually prepending/appending to strings because not all languages have the same grammar rules. Do you have an example of where this shows up? Could we just use "required" instead?

Reply:
@KevinFCormier - This is where it is located.  I changed to 'required' (the 'i18n' is just for my testing)
![Screen Shot 2022-10-18 at 2 49 21 PM](https://user-images.githubusercontent.com/10394596/196518184-f100463c-ccac-4b7f-9199-de050b0c936e.png)

Reply:
Please also remove `isRequired` from the list of strings.